src/django/api/views.py

```diff
@@ -1793,6 +1796,8 @@ def split(self, request, pk=None):
                         m.facility_list_item.source.contributor.id
                         if m.facility_list_item.source.contributor else None,
                         'match_id': m.id,
+                        'is_geocoded':
```

Comment:
It seems that the geocoded location wasn't being passed through the API properly. Because we do not actually need to know the location on the front end, just whether it exists, I've updated to send a boolean instead of the geocoded point.